modules/claims_api/app/controllers/claims_api/v1/forms/intent_to_file_controller.rb

```diff
@@ -46,17 +46,18 @@ def submit_form_0966
         # @return [JSON] Response from BGS
         def active
           check_for_type
-
-          bgs_response = bgs_service.intent_to_file.find_intent_to_file_by_ptcpnt_id_itf_type_cd(
+          message = "No Intent to file is on record for #{target_veteran_name} of type #{active_param}"
```

Comment:
Nitpick: I'd move this back to where it was at the minimum. we don't really care to call `target_veteran_name` and `active_param` if we don't have to. You may even move the `message` declaration and the `raise` in to an `if` block so neither run unless needed.